### ISSUE TITLE
Missing Height Data

### ISSUE THREAD

**Issue by brymz** [TREC-Agroecology/hemp#47]
- `PilotPlot1_summer_2019_PlantHeight.csv`
- `PilotPlot2_summer_2019_PlantHeight.csv`

**Comment by wdlngtn** [comment]
Is this what you are looking for?


https://github.com/TREC-Agroecology/hemp/blob/master/data/PilotPlot1_summer_2019_PlantHeight-SexRatio.csv

https://github.com/TREC-Agroecology/hemp/blob/master/data/PilotPlot2_summer_2019_PlantHeight-SexRatio.csv

**Comment by brymz** [comment]
Nope. Those are the individual plant height measurements. I'm looking for the canopy heights. The goal is to match height measurement across variety and planting date trials. I have canopy heights for variety trial in `VarietyTrial1_summer_2019_PlantHeight.csv`, but not for PlatingDate1 and PlantingDate2.

**Comment by wdlngtn** [comment]
We only have canopy heights for the variety trial, but I  think 10
individuals is enough to estimate the canopy.
